integration_tests/run_pyspark_from_build.sh

```diff
@@ -174,7 +174,8 @@ else
     else
         TEST_PARALLEL_OPTS=("-n" "$TEST_PARALLEL")
     fi
-    RUN_DIR=${RUN_DIR-"$SCRIPTPATH"/target/run_dir}
+
+    RUN_DIR=${RUN_DIR-$(mktemp -u -p "$SCRIPTPATH"/target -d run_dir-$(date +%Y%m%d%H%M%S)-XXXX)}
```

Comment:
Do we really want `-u` here?  Adding it means `mktemp` no longer guarantees the directory is unique.  Seems like removing it is exactly what we want, since it will not be evaluated if RUN_DIR is already set, and it will guarantee a unique directory when it does run.  If it does create the directory, the `mkdir -p` will not fail for an existing directory (which we need anyway if RUN_DIR is explicitly set to an existing directory).

Reply:
+1, I missed that -u is labeled unsafe in the help.